travvy/src/components/AttractionsList.vue

```diff
@@ -42,7 +42,7 @@
       <div v-if="$store.state.recommendedAttractions.length > 0">
       <br>
       <h3>Description</h3>
-      <p>{{$store.state.city}} is the largest city in {{$store.state.recommendedAttractions[0].country}}, and is home to attractions such as the {{$store.state.recommendedAttractions[0].attraction_name}}, {{$store.state.recommendedAttractions[1].attraction_name}}, and {{$store.state.recommendedAttractions[2].attraction_name}}.</p>
+      <p>{{$store.state.city}} is the largest city in {{$store.state.recommendedAttractions[0].country}}, and is home to attractions such as the {{$store.state.recommendedAttractions[0].attraction_name}}.</p>
```

Comment:
This would not work if the return was of length less than one

Reply:
Good idea to change it. Hopefully in future sprints we can make this writing more dynamic.

Reply:
Yes, I agree, I can add this as ticket in the backlog for the next sprint!